src/core/CMakeLists.txt

```diff
@@ -39,11 +39,12 @@ list(APPEND core_SOURCES typesinfo.c)
 list(APPEND core_SOURCES var.c)
 
 if(TODBC_WINDOWS)
+  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/taos_odbc_ver_en.rc.in" "${CMAKE_CURRENT_SOURCE_DIR}/taos_odbc_ver_en.rc")
```

Comment:
better make generated-file in build-directory but not source

Reply:
ok，i will modify the template generation source file to the build directory